### DISCUSSION TITLE
Is there an easy way to turn off logs for healthcheck paths?

### DISCUSSION THREAD

**Discussion by guyschlider** [tiangolo/uwsgi-nginx-docker#220] [Questions] [answered]
Lets say I want to add a custom location {} directive to the nginx.conf without having a fully modified version of it,
What is the best practice for doing so?
`
location /health {
                access_log off;
                error_log /dev/stderr error;
                proxy_pass http://api;
         }
`
for example

**Comment by tony-garcia** [comment, marked as answer]
This is how I turned off logging for health check requests. (in my app, the health checks were hitting the URL /heathcheck/

```
map $request_uri $loggable {
  /healthcheck/ 0;
  default 1;
}

server {
...
  access_log  /var/log/nginx/access.log  main if=$loggable;
...
}
```
  
source: http://nginx.org/en/docs/http/ngx_http_log_module.html
